Source/RapyutaSimulationPlugins/Private/Sensors/ROS2CameraActor.cpp

```diff
@@ -0,0 +1,20 @@
+// Copyright 2020-2021 Rapyuta Robotics Co., Ltd.
+
+#include "Sensors/ROS2CameraActor.h"
+
+AROS2CameraActor::AROS2CameraActor() : Super()

```

Comment:
`Super()` would be already automatically called, thus is not required for a default constructor.